scss/_custom.scss

```diff
@@ -245,54 +245,56 @@ $transition-collapse:    height .35s ease !default;
 //
 // Font, line-height, and color for body text, headings, and more.
 
-$font-family-sans-serif: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif !default;
+$font-family-sans-serif: "Helvetica Neue", Arial, Roboto, sans-serif !default;
 $font-family-monospace:  Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace !default;
 $font-family-base:       $font-family-sans-serif !default;
 
-$font-size-base: 1rem !default; // Assumes the browser default, typically `16px`
-$font-size-lg:   1.25rem !default;
-$font-size-sm:   .875rem !default;
-$font-size-xs:   .75rem !default;
+$font-size-base: .9rem;
+$font-size-lg:   1.1rem;
+$font-size-sm:   .75rem;
+$font-size-xs:   .7rem;
 
-$font-weight-normal: normal !default;
-$font-weight-bold: bold !default;
+$font-weight-light: 300;
```

Comment:
I think for any new variables you are introducing, lets add an `rc-` prefix, i.e. `$rc-font-weight-light` so that we can clearly see it and won't conflict if they add similar variable names later